.github/workflows/deploy.yml

```diff
@@ -8,27 +8,24 @@ on:
 jobs:
   deploy:
     runs-on: ubuntu-latest
-  
+ 
     steps: 
+    - name: Checkout repository
+      uses: actions/checkout@v2
+
     - name: Setup CD Environment
       uses: yetanalytics/actions/setup-env@v0
-    
+
     - name: Extract version
       id: version
       run: echo ::set-output name=VERSION::${GITHUB_REF#refs\/tags\/v}
 
-    - name: Build JAR file
-      run: make build VERSION=${{ steps.version.outputs.VERSION }}
-    
-    # This is mainly a debug step that exists to ensure sanity checking
-    - name: Publish artifact
-      uses: actions/upload-artifact@v2
+    - name: Build and deploy to Clojars
+      uses: yetanalytics/actions/deploy-clojars@1f68c6d414e8d0986c6a6554e7d0efff847ab50c
       with:
-        name: colossal-squuid-artifact-${{ steps.version.outputs.VERSION }}
-        path: target/colossal-squuid-${{ steps.version.outputs.VERSION }}.jar
+          artifact-id: 'colossal-squuid'
+          resource-dirs: '[]'
+          version: ${{ steps.version.outputs.VERSION }}
+          clojars-username: ${{ secrets.CLOJARS_USERNAME }}
+          clojars-password: ${{ secrets.CLOJARS_PASSWORD }}
```

Comment:
This speaks more to the inputs on the reusable workflow itself, but it might be better to explicitly call this a deploy token so implementing devs know to go get one of those

Reply:
`clojars-password` was originally called `clojars-token`, but I changed it since it was `CLOJARS_PASSWORD` everywhere else. It seems like you would like it to be changed back to `clojars-token` or `clojars-deploy-token`, is that right?

Reply:
Yeah, even though a password _could_ be used it just feels like better practice to call it what folks _should_ be using.